build.sh

```diff
@@ -6,30 +6,29 @@ log() {
     fi
 }
 
-if [ "$1" == "help" ]; then
-    echo "$0        | Build h-e-l-l-o-w-o-r-l-d"
-    echo "$0 clean  | Remove all build related files"
-    echo "$0 run    | Run hello_world in container"
-    echo "$0 shell  | Run shell in container"
-    echo "$0 help   | Show this help message"
-    exit
-fi
-
 if [ -z "$DONT_BUILD_CONTAINER" ]; then
     log "[*] Building container"
     log ""
 
-    docker-compose build --build-arg COMMIT="$(git log -1 --format=%h)"
+    docker-compose build
 
     log ""
 fi
 
+if [ -z "$1" ]; then CONTAINER_EXEC="time bash -e /code/start-build.sh";
```

Comment:
For consistency, may i suggest making `build` a valid option here too?

Reply:
on it

Reply:
done